jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -636,6 +667,55 @@ static JavaContainerBuilder getJavaContainerBuilderWithBaseImage(
     return entrypoint;
   }
 
+  // It's perfectly fine to always generate a new temp file or rewrite an existing file. However,
```

Comment:
nit: perhaps we could put this in a javadoc format?

Reply:
In my view, this is a code comment that so relevant as a Javadoc.

Reply:
Ah I see. In that case, let's put this under `/* ...*/` for consistency.

Reply:
Relocated the comment (which I think is a better place actually) to prevent someone from mistakenly converting it to a Javadoc when we use the `/* ... */` style. And within a method, we've been using `//`, so I think we're good.